README.md

```diff
@@ -0,0 +1,14 @@
+# Ensembl Web Resolver
+The resolver service generates new Ensembl website urls for different features based on their stable ids, as well as other optional parameters.
+### Deploy the app and run docker-compose:
+ 
+ `$ git clone https://github.com/Ensembl/ensembl-web-resolver.git`
+ `$  cd ensembl-web-metadata-api`
+ `$ mv sample-env .env`
```

Comment:
Should we replace the search hub url in `sample-env` with beta url?

Reply:
`https://beta.ensembl.org/api/search/stable-id`
with below data as post request
{code}
{
    "stable_id": "WBGene00016374",
    "type": "gene"
}
{code}

Reply:
What do you want me to do with the payload?

Reply:
env-sample is updated

Reply:
Nothing, that payload was just for you to try out :-)  Approving now